shanoir-ng-users/src/main/java/org/shanoir/ng/accessrequest/controller/AccessRequestApi.java

```diff
@@ -19,107 +19,108 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 
-import io.swagger.annotations.Api;
-import io.swagger.annotations.ApiOperation;
-import io.swagger.annotations.ApiParam;
-import io.swagger.annotations.ApiResponse;
-import io.swagger.annotations.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
 
 /**
  * Api for access request, to make a demand on 
  * @author jcome
  *
  */
-@Api(value = "accessrequest")
+@Tag(name = "accessrequest")
 @RequestMapping("/accessrequest")
 public interface AccessRequestApi {
 
-	@ApiOperation(value = "", notes = "Saves a new access request", response = AccessRequest.class, tags = {})
+	@Operation(summary = "", description = "Saves a new access request")
 	@ApiResponses(value = {
-			@ApiResponse(code = 200, message = "created access request", response = AccessRequest.class),
-			@ApiResponse(code = 401, message = "unauthorized", response = AccessRequest.class),
-			@ApiResponse(code = 403, message = "forbidden", response = AccessRequest.class),
-			@ApiResponse(code = 422, message = "bad parameters", response = AccessRequest.class),
-			@ApiResponse(code = 500, message = "unexpected error", response = AccessRequest.class) })
+			@ApiResponse(responseCode = "200", description = "created access request"),
```

Comment:
Shouldn't we still define the response type, so that the schema is displayed in swagger ?
Or is it now deducted in swagger v3 ?

Reply:
swagger v3 removed the response type